### PR TITLE
README uses 1.9 hash style

### PR DESCRIPTION
**Pull request by tibbon**
Updates the hash style from hashrockets to the 1.9+ hash style. The project requires 1.9.3+ so this should be valid syntax for all users and more friendly/consistent for developers.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1123,39 +1123,39 @@ Job.aasm.states.map(&:name)
...
 
 # show all possible events except a specific one
-job.aasm.events(:reject => :sleep).map(&:name)
+job.aasm.events(reject: :sleep).map(&:name)
 #=> [:clean]
 
 # list states for select
 Job.aasm.states_for_select
-=> [["Sleeping", "sleeping"], ["Running", "running"], ["Cleaning", "cleaning"]]
+#=> [["Sleeping", "sleeping"], ["Running", "running"], ["Cleaning", "cleaning"]]
```
Comment (anilmaurya) [inline README.md]: why commented this line ?

Comment (tibbon) [reply README.md]: I was hoping to bring it more in line with the prior examples, which used a comment to denote it was a returned value. This one wasn't consistent with that.

Comment (anilmaurya) [reply README.md]: ok 👍

**Comment by anilmaurya** [comment]
Thank you @tibbon .

**Pull request merged**